.devcontainer/dev/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM registry.opensuse.org/opensuse/leap:15.5
+RUN zypper ref && \
+      zypper -n install \
+      java-17-openjdk-devel \
+      openssh \
+      rsync \
+      apache-ivy \
+      ant \
+      ant-junit5 \
```

I had issues with this when building the server all in one container due to a dependency not being found with the default repos. I looked at OBS and used the following instead: 
```suggestion
      ant-junit \
```

---

That was the error, I got:

```bash
 > [ 5/11] RUN zypper --non-interactive install       java-17-openjdk-devel       openssh       rsync       apache-ivy       ant       ant-junit5       servletapi5       cpio       spacecmd       expect       wget:
1.898 No update candidate for 'servletapi5-5.0.18-2.39.noarch'. The highest available version is already installed.
1.898 'spacecmd' is already installed.
1.898 No update candidate for 'spacecmd-5.0.8-230900.1.1.uyuni3.noarch'. The highest available version is already installed.
1.898 Resolving package dependencies...
1.953 
1.953 Problem: 1: nothing provides 'mvn(org.opentest4j:opentest4j)' needed by the to be installed junit5-5.10.2-150200.3.10.3.noarch
1.953  Solution 1: do not install ant-junit5-1.10.14-150200.4.28.1.noarch
1.953  Solution 2: break junit5-5.10.2-150200.3.10.3.noarch by ignoring some of its dependencies
1.953 
1.953 Choose from above solutions by number or cancel [1/2/c/d/?] (c): c
------
```